prac_09/unreliable_car_test.py

```diff
@@ -0,0 +1,25 @@
+"""
+CP1404
+Test program that shows the unreliable_car subclass of Car.
+"Reliability" is a float between 0 and 100, that represents the percentage chance that
+the drive method will actually drive the car.
+"""
+
+
+from prac_09.unreliable_car import UnreliableCar
+
+
+def main():
+    """Demo test code to show how to use taxi subclass."""
+    my_shitbox = UnreliableCar("Toyota Echo", 100, 80.0)
```

Comment:
Might need changing before a learnJCU submission. Swearing is a sin!!!!!
![NerdNerdyGIF](https://github.com/ThomasMehes/cp1404practicals/assets/99230734/c91334d0-0f6e-4809-b717-edeba7f20a31)

Reply:
As an Australian this would be the most correct terminolgy, particularly because I gave my cars details, but you're right.